definitions/ext-ddmysql/golden_metrics.yml

```diff
@@ -0,0 +1,1462 @@
+bufferPoolDirty:
```

Comment:
There are a lot of metrics here.. i would feel too many? How do we internally handle GM limits @IreneMLC ?

Reply:
Yes, we have a limit of 10 GMs per entity type ([docs](https://github.com/newrelic-experimental/entity-synthesis-definitions/blob/main/docs/golden_metrics.md#golden-metrics)) but it's not enforced on the repo's check, I'll look into adding it.

Reply:
@austinNester You could create a dashboard with all of these metrics. Ideally, we should only have a small handful of golden metrics (favourite metrics) approx 4 should be ideal. as Irene says, 10 is the limit